components/GlobalSpinner/GlobalSpinner.test.tsx

```diff
@@ -41,5 +41,6 @@ describe('GlobalSpinner', () => {
       'global-spinner-image',
     ) as HTMLImageElement;
     expect(image.src).toContain('global-spinner');
+
```

Comment:
nit: remove this.

Reply:
Remove the `expect` or the `as HTMLImageElement?